src/h3lib/lib/coordijk.c

```diff
@@ -66,7 +66,7 @@ void _hex2dToCoordIJK(const Vec2d *v, CoordIJK *h) {
     a2 = fabsl(v->y);
 
     // first do a reverse conversion
-    x2 = a2 / M_SIN60;
+    x2 = a2 * M_RSIN60;
     x1 = a1 + x2 / 2.0;
```

Comment:
I just spotted this. I'm not sure if it matters since powers of two are quick anyway, but I figured I would document that we could change it to `x1 = a1 + x2 * 0.5;`
Or since this is the only usage of `M_RSIN60`, just craft a `M_RSIN60_DIV_BY_2`

Reply:
If the same assembly is produced it sounds like it's fine to leave as-is because compiler optimizations take care of it for us?

Reply:
Yes, any compiler at -O1 or higher opt figures it out.